src/main/java/uk/gov/ons/ctp/response/casesvc/service/impl/CaseServiceImpl.java

```diff
@@ -191,31 +191,60 @@ public CaseEvent createCaseEvent(final CaseEvent caseEvent, final Case newCase,
       // should we create an ad hoc action?
       createAdHocAction(category, caseEvent);
 
-      // transition case group status
-      CaseGroup caseGroup = caseGroupRepo.findOne(targetCase.getCaseGroupFK());
-      try {
-        caseGroupService.transitionCaseGroupStatus(caseGroup, caseEvent.getCategory(), targetCase.getPartyId());
-      } catch (CTPException e) {
-        //The transition manager throws an exception if the event doesn't cause a transition, however there are lots of
-        // events which do not cause CaseGroupStatus transitions, (this is valid behaviour).
-        log.debug(e.getMessage());
-      }
+      transitionCaseGroupStatus(targetCase, caseEvent);
 
       // if this is a respondent enrolling event
-      if (caseEvent.getCategory().toString().equals("RESPONDENT_ENROLED")) {
+      if (caseEvent.getCategory().equals(CategoryDTO.CategoryName.RESPONDENT_ENROLED)) {
         // are there other case groups that need updating
         List<CaseGroup> caseGroups = caseGroupService.transitionOtherCaseGroups(targetCase);
```

Comment:
Not from this PR but this function name is a bit misleading. `caseGroupService.transitionOtherCaseGroups` doesn't actually transition anything it just retrieves casegroups